packages/react/src/lib/whatInput.ts

```diff
@@ -82,45 +82,45 @@ const setUp = () => {
   // add correct mouse wheel event mapping to `inputMap`
   inputMap[detectWheel()] = 'mouse'
 
-  addListeners()
+  addListeners(window)
```

Comment:
Thinking from a side of simplifying things, curious if it would make sense to add listeners initially to `document` as for these types of events there is no difference `document` or `window`. And then in your method attach additional documents as you do. So we'll refer only to document object and won't diversify between window and document

Reply:
I think that reason to use `window` is conditions like this:
```
if (eventTarget.PointerEvent) {
} else if (window.MSPointerEvent) {
```

Reply:
we decided to do the opposite - always attach them to window (or to document.defaultView which is also a Window)